src/map-interactions.react.js

```diff
@@ -167,6 +174,9 @@ export default class MapInteractions extends Component {
     const pos = this._getMousePos(event);
     this.setState({pos});
     this.props.onMouseUp({pos});
+    if (!this.state.dragged) {
+      this.props.onMouseClick({pos});
```

Comment:
`isDragging` is set to true on mouse down, so it will not help telling whether it's been a click on mouse up. I don't want to change this behavior, in case it breaks other things.